build/Dockerfile

```diff
@@ -157,6 +157,11 @@ ENV OPENFLOW_HEADERS='include/openflow/intel-ext.h \
     include/openflow/openflow-common.h \
     include/openflow/openflow.h '
 RUN echo "5--- installing openvswitch dependancies ---" && \
+    rm -f /tmp/get-pip.py > /dev/null 2>&1 && \
+    apt-get install -y python2.7 && \
+    wget https://bootstrap.pypa.io/pip/2.7/get-pip.py -O /tmp/get-pip.py && \
+    python2.7 /tmp/get-pip.py && \
+    pip2 install six && \
```

Comment:
I kept getting permission denied error, got tired of it and tacked sudo infron of all of them.

Reply:
Added in next commit